src/main/scala/xiangshan/mem/pipeline/AtomicsUnit.scala

```diff
@@ -268,65 +270,60 @@ class AtomicsUnit(implicit p: Parameters) extends XSModule with MemoryOpConstant
           state := s_cache_req
         }
       } .otherwise {
-        // latch response
-        dcache_resp_data := io.dcache.resp.bits.data
-        dcache_resp_id := io.dcache.resp.bits.id
-        dcache_resp_error := io.dcache.resp.bits.error
-        state := s_cache_resp_latch
-      }
-    }
-  }
-
-  when(state === s_cache_resp_latch) {
```

Comment:
For timing reasons, `s_cache_resp_latch` should not be removed

Reply:
OK, I'll change it back